packages/loot-core/src/server/budget/goaltemplates.ts

```diff
@@ -218,7 +233,7 @@ async function processTemplate(
       }
     }
     if (budgeted) {
-      if (!force) {
+      if (!force && !category.payDistributeTemplateActive) {
```

Comment:
Since Pay distribution is designed to be done regularly allowing it to be done for the whole month without overwriting what was done to other categories i felt was important.